### DISCUSSION TITLE
400 BadRequest 단위 테스트 작성법

### DISCUSSION THREAD

**Discussion by verus-j** [woowacourse-teams/2022-moamoa#135] [토론]
현재 RestAssured를 활용한 인수 및 통합 테스트에 BadRequest 케이스의 테스트가 모여 있음...
Bad Request의 경우 앞 부분만 확인하면 되기에 리소스가 비싼 RestAssured가 아닌 WebMvcTest로 가볍게 돌려도 충분히 검증이 가능해 보임
RestAssured에서 작성된 Bad Request 테스트를 WebMvcTest로 옮겨 가볍게 돌리면 좋을 것으로 보임

각자 의견 달아주세요~

**Comment by jaejae-yoo** [comment]
저도 동의합니다 ~
테스트가 많아지면 너무 무거워질 것 같아요.
400 예외 테스트는 WebMvcTest로 돌려도 충분히 검증될 것 같아요!
